lib/THNN/generic/SpatialMaxPooling.c

```diff
@@ -61,18 +61,8 @@ static void nn_(SpatialMaxPooling_updateOutput_frame)(real *input_p, real *outpu
   }
 }
 
-static int nn_(SpatialMaxPooling_updateOutput)(lua_State *L)
+void THNN_(SpatialMaxPooling_updateOutput)(THNNState *state, THTensor *input, THTensor *output, THTensor *indices, int kW, int kH, int dW, int dH, int padW, int padH, int ceil_mode)
```

Comment:
Is ceil_mode a boolean value? Maybe we could use type bool? It is btw interesting that indices historically is a floating point tensor (may become a problem for indices > 2^23 (size of float mantissa)).